orientdb/src/main/scala/akka/stream/alpakka/orientdb/OrientDBSinkSettings.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.orientdb
+
+import com.orientechnologies.orient.core.db.OPartitionedDatabasePool
+
+//#sink-settings
+final case class OrientDBSinkSettings(oDatabasePool: OPartitionedDatabasePool,
+                                      maxPartitionSize: Int = Runtime.getRuntime.availableProcessors(),
+                                      maxPoolSize: Int = -1,
+                                      maxRetry: Int = 1,
+                                      retryInterval: Int = 5000,
+                                      bufferSize: Int = 10)
+//#sink-settings
+{
+
+  def this(oDatabasePool: OPartitionedDatabasePool) =
```

Comment:
This constructor will cause issues with the default constructor because scala won't be able tu figure out which one to use when having `OrientDBSinkSettings(myODatabasePool)`

It shouldn't be here.
If you need for the Java API, just add a method call `create` as other connectors do.
```
def create(oDatabasePool: OPartitionedDatabasePool): OrientDBSinkSettings = this(oDatabasePool)
```